apps/root/src/pages/token-profile/header/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import useNetWorth from '@hooks/useNetWorth';
+import { Token } from 'common-types';
+import TokenIcon from '@common/components/token-icon';
+import { ContainerBox, Typography, colors } from 'ui-library';
+import NetWorthNumber from '@common/components/networth-number';
+
+interface TokenProfileHeaderProps {
+  token?: Token;
```

Comment:
We can assume token is never undefined right?

Reply:
we could, in case we reuse this I'll prefer to keep it optional for now